### PR TITLE
Adding grammar checker script

### PR DESCRIPTION
**Pull request by rajatkeshri**
## Type of change

- [x] New Script (non-breaking change which adds a new script to the repository)

## Has This Been Tested?

- [x] I confirm that I have tested my script locally and verified that it works as intended.

## Checklist:

- [x] I have gone through the contributing guidelines before making this Pull Request.
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: scripts/Text Grammar Corrector/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+## About :
+
+---
+
+- This script takes an englsh text paragraph input. Using the language API, it identifies the mistakes in the paragraph. The script then uses the mistakes dictionary and returns the updated paragraph.
+
+---
+
+## How to use it :
```
Comment (rijusougata13) [inline scripts/Text Grammar Corrector/README.md:11]: hi @rajatkeshri  please mention the detailed process to install requirements and run the python file. mention the command to install and run the file

**Review by rijusougata13 — CHANGES_REQUESTED** [review]

File: scripts/Text Grammar Corrector/grammer bot.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# Imports
+import language_tool_python
+
+# Initial API
+tool = language_tool_python.LanguageToolPublicAPI("en-US")
+
+# initalizae filename
+FILENAME = "input.txt"
+
+text = """ update your text here. tool will will fidn grammatically isues and fix it. you can use this text too see an few of of the problems that LanguageTool can detecd. What do you thinks of grammar checkers? Please not that they are not perfect."""
```
Comment (OBITORASU) [inline scripts/Text Grammar Corrector/grammer bot.py:10]: `fidn` should be find

File: scripts/Text Grammar Corrector/grammer bot.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
+# Imports
+import language_tool_python
+
+# Initial API
+tool = language_tool_python.LanguageToolPublicAPI("en-US")
+
+# initalizae filename
+FILENAME = "input.txt"
```
Comment (OBITORASU) [inline scripts/Text Grammar Corrector/grammer bot.py:8]: I think it is better that we ask the user for input either during runtime or with command line arguments using `argparse`.

**Comment by OBITORASU** [comment]
Check the comments

File: scripts/Text Grammar Corrector/grammer bot.py (hunk 2 above)
Comment (rajatkeshri) [reply scripts/Text Grammar Corrector/grammer bot.py:10]: The text is written with bugs on purpose. The script would correct it

File: scripts/Text Grammar Corrector/README.md (hunk 1 above)
Comment (rajatkeshri) [reply scripts/Text Grammar Corrector/README.md:12]: Updated

File: scripts/Text Grammar Corrector/grammer bot.py (hunk 3 above)
Comment (rajatkeshri) [reply scripts/Text Grammar Corrector/grammer bot.py]: Done

**Review by OBITORASU — CHANGES_REQUESTED** [review]
See the review below forgot to add the code comment here.

File: scripts/Text Grammar Corrector/grammer bot.py [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+from pathlib import Path
+
+# ------------------------------------------------------
+# Initialize API
+tool = language_tool_python.LanguageToolPublicAPI("en-US")
+
+# get the input text file from the user
+parser = argparse.ArgumentParser()
+parser.add_argument("input_file_path", type=Path)
+p = parser.parse_args()
```
Comment (OBITORASU) [inline scripts/Text Grammar Corrector/grammer bot.py:13]: Nit pick. Can you add a -h flag for usage/help and make the file path argument mandatory for the script to run?

**Review by OBITORASU — CHANGES_REQUESTED** [review]
Left a review

**Comment by rijusougata13** [comment]
LGTM!

File: scripts/Text Grammar Corrector/grammer bot.py (hunk 4 above)
Comment (rajatkeshri) [reply scripts/Text Grammar Corrector/grammer bot.py]: Done

**Comment by OBITORASU** [comment]
LGTM!

**Pull request merged**